src/Sulu/Bundle/ContentBundle/DependencyInjection/Configuration.php

```diff
@@ -53,6 +48,26 @@ public function getConfigTreeBuilder()
                         ->booleanNode('websocket')->defaultFalse()->end()
                     ->end()
                 ->end()
+                ->arrayNode('collaboration')
+                    ->canBeEnabled()
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('interval')
+                            ->defaultValue(300000)
+                            ->info('Defines the interval in milliseconds between the keep alive messages for the collaboration feature')
+                        ->end()
+                        ->scalarNode('threshold')
+                            ->defaultValue(10000)
+                            ->info('Defines the threshold after which the collabaration without keep alive signal is considered finished')
+                        ->end()
+                        ->scalarNode('entity_cache')
```

Comment:
we should think about "centralizing" out caches (for all places in sulu) with a cache library like http://www.stashphp.com/Symfony.html and use everywhere the same cache service with different namespaces for the entry. then the developer can choose the implementations and stack them at one single place.